ceno_zkvm/src/tables/program.rs

```diff
@@ -104,11 +104,22 @@ pub struct ProgramTableConfig {
     mlt: WitIn,
 }
 
-pub struct ProgramTableCircuit<E, const PROGRAM_SIZE: usize>(PhantomData<E>);
+#[derive(Clone, Default)]
```

Comment:
Default here is 0 size which is probably not what you want. Just leave it without default to force the use of the `…_param` function?